docs/pages/accounts/sso.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Single Sign-On (SSO)
+description: Learn about how your enterprise organization can use your Identity Provider to manage Expo users.
+---
+
+## Configure SSO for your organization
+
+SSO is available for Enterprise Plan customers. To get started, contact us and we'll help you set it up for your organization.
+
+> The Expo SSO system should support any Identity Provider (IdP) that supports [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html). We have tested [Okta](https://www.okta.com/) and plan to test others as requested.
```

Comment:
```suggestion
> The Expo SSO system supports any Identity Provider (IdP) that supports [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html). We have tested [Okta](https://www.okta.com/) and plan to test others as requested.
```

"should support" is not the phrase I'd like us to use in docs. It signifies that we are confident but not sure.

Reply:
Wonder if we could rephrase this to the effect of "We currently formally support Okta, but we implement the OIDC Discovery 1.0 spec, and are working to verify additional providers that implement the spec. If you use another provider and are interested in SSO, let us know."